### PR TITLE
cmd/gentypes: sync at DAP 1.59

### PR DESCRIPTION
**Pull request by tamayika**
Updated to the latest version.

### CODE REVIEW

File: cmd/gentypes/gentypes.go [hunk 1]
```diff
@@ -107,7 +107,15 @@ func parsePropertyType(propValue map[string]interface{}) string {
 			if !ok {
 				log.Fatal("missing additionalProperties field when type=object:", propValue)
 			}
-			valueType := parsePropertyType(additionalProps.(map[string]interface{}))
+			var valueType string
```
Comment (tamayika) [inline cmd/gentypes/gentypes.go:110]: There is `additonalProperties: true` in `StartDebuggingRequestArguments.configuration`.
So I added type swtich to cover this situation.

**Comment by tamayika** [comment]
@suzmue Can you review this PR?

**Review by suzmue — APPROVED** [review]
Thanks! Looks good to me.

**Review by findleyr — APPROVED** [review]

**Review by jamalc — APPROVED** [review]

**Comment by tamayika** [comment]
@suzmue @jamalc 
Please merge this PR and release new version.

**Pull request merged**